Contacts/People/modules.mixer

```diff
@@ -499,9 +499,10 @@
   </li>
 
   <li> <strong>Structure Descriptions for Finite Groups</strong>
-       <em> &nbsp;&nbsp;<br/>by Stefan Kohl, Markus P�schel and
+       <em> &nbsp;&nbsp;<br/>by Stefan Kohl, Markus Püschel and
        Sebastian Egner,<br />
        maintained by 
+       <mixer person="Gabor Horvath" data="name_link_city"/> and
```

Comment:
Please edit this PR leaving only one change which adds the line `<mixer person="Gabor Horvath" data="name_link_city"/>`. First of all, it's not a good practice to mix in one commit actual editing with changing encoding/whitespaces/formatting etc. Second, it's GitHub displaying umlauts in the wrong way, but when Mixer builds html from the source, they all look correctly. I've fetched your PR onto my machine (thanks to @fingolfin's tip about `fetch-pr`) and built the website with your changes and then all umlauts are not displayed properly.